.github/workflows/publish.yml

```diff
@@ -94,7 +94,7 @@ jobs:
                   name: release
                   path: dist/
 
-            - uses: softprops/action-gh-release@v2.0.6
+            - uses: softprops/action-gh-release@v2
```

Comment:
I've unpinned the softprops/actions-gh-release action.
Pinning this action to specific patch version just creates noise from dependabot. Given that we currently don't pin any other actions to the exact versions, let's unpin this one as well.

Reply:
Same as in https://github.com/aiidalab/aiidalab-docker-stack/pull/489